Waiter.py

```diff
@@ -26,37 +26,64 @@
 OrderList = list(ItemDet.keys())
 ItemP = list(ItemDet.values())
 
+ccount = 0
+fcount = 0
+tcount = 0
+scount = 0
+
+#Make list for VALUES through user input
+OrderQuant = [ccount, fcount, tcount, scount]
+
```

Comment:
if you make it like this, you will need to edit your code whenever you add an item in the menu. How about making a dictionary using the name of the order as the key and the count of the order as the value?